data/types/bolus/bolus.go

```diff
@@ -65,6 +79,12 @@ func (b *Bolus) Normalize(normalizer data.Normalizer) {
 	if b.InsulinFormulation != nil {
 		b.InsulinFormulation.Normalize(normalizer.WithReference("insulinFormulation"))
 	}
+	if b.InsulinOnBoard != nil {
+		b.InsulinOnBoard.Normalize(normalizer)
+	}
```

Comment:
according to the spec:
insulinOnBoard: amount of active insulin estimated by the system. **This field will be accepted when prescriptor is either auto or hybrid. It will be ignored for manual entries.**

I don't see the last part in the normalizer.